libs/designsystem/src/lib/components/checkbox/checkbox.component.scss

```diff
@@ -12,6 +13,23 @@ $default-checkbox-radio-size: map.get(utils.$checkbox-radio-sizes, 'md');
   @return ($target-height - $checkbox-icon-size) * 0.5;
 }
 
+@function _get-hover-color($variant, $loudness: interaction-state.$default-loudness, $flip: false) {
+  $_background: utils.get-color($variant, $getValueOnly: true);
+
+  $_lightness: 0% !default;
+
+  @if $flip {
+    $_lightness: utils.get-loudness($loudness);
+  } @else {
+    // negative number means "make it darker"
+    $_lightness: -1 * utils.get-loudness($loudness);
+  }
+
+  $hover-color: #{color.scale($_background, $lightness: $_lightness)};
+
+  @return $hover-color;
+}
```

Comment:
I'm thinking, maybe this should be generalised and moved to the shared interaction state module.
What does the reviewer think?

Reply:
Since it is used in #2128 I fully agree!